chain/factory/outputfactory/output.cue

```diff
@@ -1,4 +1,4 @@
-package output
+package outputfactory
```

Comment:
outputfactory is not a good name.. We have output.cue everywhere.
This should be named as statewriter.